libres/lib/enkf/local_updatestep.cpp

```diff
@@ -36,25 +36,22 @@
 struct local_updatestep_struct {
     UTIL_TYPE_ID_DECLARATION;
     char *name;
-    vector_type *ministep;
+    std::vector<std::unique_ptr<local_ministep_type>> ministep;
 };
 
 UTIL_SAFE_CAST_FUNCTION(local_updatestep, LOCAL_UPDATESTEP_TYPE_ID)
 
 local_updatestep_type *local_updatestep_alloc(const char *name) {
-    local_updatestep_type *updatestep =
-        (local_updatestep_type *)util_malloc(sizeof *updatestep);
+    local_updatestep_type *updatestep = new local_updatestep_type();
 
     UTIL_TYPE_ID_INIT(updatestep, LOCAL_UPDATESTEP_TYPE_ID);
     updatestep->name = util_alloc_string_copy(name);
-    updatestep->ministep = vector_alloc_new();
-
+    updatestep->ministep = std::vector<std::unique_ptr<local_ministep_type>>();
```

Comment:
This line does nothing and can be removed.

Reply:
thought so to, Done